doc/nrf/app_dev/device_guides/coprocessors/index.rst

```diff
@@ -7,17 +7,18 @@ Developing with coprocessors
 
    The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 device.
 
-The following pages provide a detailed guide on utilizing coprocessor effectively within the nRF54L15 platform.
-
-The VPR coprocessor (Fast Lightweight Peripheral Processor - FLPR) can be utilized in two distinct ways:
+You can use the VPR coprocessor (Fast Lightweight Peripheral Processor - FLPR) as follows:
```

Comment:
Keep the mention of the hardware platforms this section is designated for. VPR is also on nRF54H20, so it's not only nRF54L.